cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/MapReduceProgramRunnerTest.java

```diff
@@ -312,9 +314,18 @@ public void testMapReduceDriverResources() throws Exception {
 
   @Test
   public void testMapreduceWithObjectStore() throws Exception {
-    final ApplicationWithPrograms app = deployApp(AppWithMapReduceUsingObjectStore.class);
+    // Deploy apps to another namespace and test cross-namespace access meanwhile
+    final ApplicationWithPrograms app = deployApp(NamespaceId.fromString("namespace:someOtherNameSpace").toId(),
```

Comment:
@anwar6953 because
1. It is unintuitive. How do someone know use a `Ids` class to create instance of `NamespaceId`?
2. It is pointless to have static method in an unrelated class (`Ids`) to do exactly the same thing as the constructor on the actual class (`NamespaceId`).